mentoring/mentoring.py

```diff
@@ -370,10 +446,14 @@ def handleAssessmentSubmit(self, submissions, suffix):
             'num_attempts': self.num_attempts,
             'step': self.step,
             'score': score.percentage,
-            'correct_answer': score.correct,
-            'incorrect_answer': score.incorrect,
-            'partially_correct_answer': score.partially_correct,
-            'message': message
+            'correct_answer': len(score.correct),
+            'incorrect_answer': len(score.incorrect),
+            'partially_correct_answer': len(score.partially_correct),
```

Comment:
@e-kolpakov Maybe. Let me see how well I can cut down the review template according to your other note and if so, that may be the better option. I did the len() check here to reduce the amount of code that was flying around in that now convoluted results template.